### ISSUE TITLE
First Meeting: Aug 11, 2015

### ISSUE THREAD

**Issue by srl295** [nodejs/Intl#8]
I don't have a "normal" week until Aug 10. So I tried to pick slots that work work for probably an "every other week" type of slot.  Please suggest more slots (on the form) if none of these work. 

http://doodle.com/ns539akkrkikf3gd

 @caridy @jasnell @KevinMartin @mhdawson @rxaviers @rwaldron @stevenatkin 

While we're at it — we'll keep minutes in the repo of course, but suggestions for the logistics of communication? Mailing list, slack, group, IRC? @mikeal 's awesome new nodejs based conferencing server? I'll default to a conf # I have and google docs if nothing else is suggested.

**Comment by kevinmartin** [comment]
I like Slack. Downfall is that messages get deleted after 10,000 messages, I believe. Not sure if that's a problem, though.

Please, lets not do a mailing list. Those are annoying and sometimes hard to follow.

Other option we can do is a [Google Hangout](https://support.google.com/hangouts/answer/3111943?hl=en)

**Comment by srl295** [comment]
@KevinMartin let's try it https://node-intl.slack.com

**Comment by srl295** [comment]
_update_ I sent out an invite for [9:00 AM PT this Tuesday](http://time.is/0900AM_11_Aug_2015_in_PT?Intl_WG) - please contact me / reply here if you haven't received the invite.

**Comment by srl295** [comment]
_update_ meeting agenda here: https://goo.gl/0LNd1n

**Comment by snostorm** [comment]
I might try to very passively listen in while working. Is it going to be recorded as a hangout? Or are you using conference #?

**Comment by snostorm** [comment]
Looks like I missed the window to join at the start. No worries.

**Comment by diagramatics** [comment]
Clarification purposes, was this on August 10th or 11th?

**Comment by srl295** [comment]
@diagramatics it was at [09:00AM on Tuesday, August 11, 2015 Pacific Time](http://time.is/0900AM_11_Aug_2015_in_PT?Intl_WG) (link for more  zones)

**Comment by diagramatics** [comment]
Awesome. Might wanna update the title to reflect the right date then :)

**Comment by srl295** [comment]
- [ ] Todo: archive notes

**Comment by srl295** [comment]
First meeting was a while ago- closing this.
